### PR TITLE
DRalgo 1.2 update

### PR DESCRIPTION
**Pull request by AndreasEkstedt**
Bug fixes:

Fixed a bug where Normalization4D would give an inconsistent result for non-abelian gauge groups
Fixed a bug where 2-loop tadpoles would fail to compute for models with many particles

Changes:

A new function "CalculatePotential[]" has been added with the exact same behavior as CalculatePotentialUS[]  
Mode->0 now give results for re-summed couplings and 2-loop masses; both in the soft and ultra-soft theory
PrintScalarMass[] now prints the full result, with PrintScalarMass["LO"] etc working as before
->The same change has been made for PrintDebyeMass,PrintPressure, PrintScalarMassUS

An "AutoRG" option has been added (default set to True). When set to False  beta-functions are not automatically added to the NLO contribution of PrintScalarMass.
Added a function BetaFunctions3DS[] which prints beta-functions in the soft theory.

### CODE REVIEW

File: EffPot.m [hunk 1]
```diff
@@ -259,6 +280,48 @@
 	];
 
 
+];
+
+(*
+	Calculates the effective potential. (just a copy of CalculatePotentialUS)
+*)
+CalculatePotential[OptionsPattern[]]:=Module[{},
```
Comment (pschicho) [inline EffPot.m:288]: Should this really be a hard copy?

File: DRalgo.m [hunk 2]
```diff
@@ -38,7 +38,7 @@
 	TexFor["DRDRDRDRDRDRDRDRDRDRDRDRDRDR "],
 	TexFor["DRalgo"],
 	TexFor[" DRDRDRDRDRDRDRDRDRDRDRDRDRDRD"]}]];
-AppendTo[result,Row[{"Version: "//TexFor,"1.1 (11-10-2023)"//TexFor}]];
```
Comment (pschicho) [inline DRalgo.m:41]: Put latest date.

File: EffPot.m (hunk 1 above)
Comment (AndreasEkstedt) [reply EffPot.m:288]: Well they do the exact thing. The user can always define what mode they want beforehand. Optimally I would just rename the CalculatePotential function altogether, but might mess up with some people that already have  CalculatePotentialUS in their scripts.

File: HardToSoft.m [hunk 3]
```diff
@@ -1159,25 +1158,19 @@
 	Temp1C=Contract[YsffC,\[Mu]IJF,FHelp,{{2,4},{5,6},{3,7}}]//SimplifySparse;
 	Contri6=-(D-2)(IF1p IF2p-IF2p I1p)(Temp1+Temp1C)//SimplifySparse;
 
-
 (*Counterterm contribution*)
-	ContriCTS=-1/2 I1p \[Epsilon]^-1*( TensorContract[Z\[Lambda]ijk,{{2,3}}]+1/2*Contract[\[Gamma]ij,\[Lambda]3,{{2,3},{4,5}}])//SparseArray//SimplifySparse;
-
+	ContriCTS=-1/2 I1p \[Epsilon]^-1*( TensorContract[Z\[Lambda]ijk,{{2,3}}]+1/2*TensorContract[\[Gamma]ij . \[Lambda]3,{2,3}])//SparseArray//SimplifySparse;
```
Comment (pschicho) [inline HardToSoft.m:1162]: Is TensorContract a new function?

File: examples/.DS_Store
Comment (pschicho) [inline examples/.DS_Store:1]: Shouldn't this file be removed?

File: EffPot.m (hunk 1 above)
Comment (pschicho) [reply EffPot.m:288]: I rather meant that one of the two instances of the function should be derived from the other to not cause copying issues in the future.

File: HardToSoft.m (hunk 3 above)
Comment (AndreasEkstedt) [reply HardToSoft.m:1162]: Yes, but it's a private function

File: DRalgo.m (hunk 2 above)
Comment (AndreasEkstedt) [reply DRalgo.m:41]: done

File: EffPot.m (hunk 1 above)
Comment (AndreasEkstedt) [reply EffPot.m:274]: Ah fair enough. I changed it now

**Review by pschicho — APPROVED** [review]
All tests passed without new error messages.

**Pull request merged**